services/brig/deb/opt/brig/templates/team/en/email/new-team.html

```diff
@@ -0,0 +1,123 @@
+  margin: 0;
+  padding: 0;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.5em;
+  color: #42474A;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+">
+  <br/>
+  <table width="100%" cellpadding="0" cellspacing="0">
+    <tr>
+      <td>
+        <center>
+          <table cellpadding="0" cellspacing="10" style="
+            max-width: 500px;
+            margin: 12px auto 0 auto;
+            font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif;
+          ">
+            <thead>
+              <tr>
+                <td style="padding-bottom: 0px;">
+                  <a href="https://wire.com">
+                    <img src="https://wire.com/p/img/email/logo-email-black.png" width="57" height="18" alt="Wire" />
+                  </a>
+                </td>
+                <td style="text-align: right; text-transform: uppercase; font-size: 12px; font-weight: 500;">
+                  <a href="https://wire.com" style="text-decoration: none; white-space: nowrap; color: #42474A;">wire.com</a>
+                </td>
+              </tr>
+            </thead>
+            <tbody style="font-size: 16px;">
+              <tr>
+                <td width="500" colspan="2" style="padding-top: 48px; padding-bottom: 0px;">
+
+                  <h1 style="font-size: 32px; font-weight: 300; text-align: center; line-height: 1.3;">
+                    Your new team
+                  </h1>
+                  <p style="color: #42474A; margin-top: 48px;">
+                    You have just created a team called
+                    <strong>${team-name}</strong> on Wire with <strong>${email}</strong>.
+                  </p>
+                  <p style="color: #42474A; margin-top: 24px; margin-bottom: 48px;">
+                    Your 30–day free trial starts today.
+                  </p>
+                  <hr style="border: 0; border-top: 1px solid #E9EDF0;">
+                  <p style="color: #42474A; margin-top: 48px;">
+                    Wire combines strong encryption, a rich feature set and ease-of-use in one app like never before.
+                  </p>
+                  </p>
+
+                  <p style="text-align: center;">
+                    <!--[if mso]>
+                      <v:roundrect href=${new-team-manage}" xmlns:v="urn:schemas-microsoft-com:vml"
+                        xmlns:w="urn:schemas-microsoft-com:office:word"
+                        style="height:48px;
+                        v-text-anchor:middle;
+                        width: 200px;"
+                        arcsize="15%"
+                        stroke="f"
+                        fillcolor="#018ED6"
+                      >
+                      <w:anchorlock/>
+                      <center>
+                    <![endif]-->
+                    <a href="${new-team-manage}" style="
```

Comment:
Where is this link meant to point to?

Reply:
@Yorgos-V? It should be https://teams.wire.com/login/ I guess